index.js

```diff
@@ -27,44 +92,40 @@ module.exports = (oppiabot) => {
     // This condition checks whether the owner account is included in
     // the whitelisted accounts.
     if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
-      await apiForSheetsModule.checkClaStatus(context);
-      await checkPullRequestLabelsModule.checkChangelogLabel(context);
-      await checkPullRequestBranchModule.checkBranch(context);
-      await checkWIPModule.checkWIP(context);
+      await runChecks(context, 'opened');
     }
   });
 
   oppiabot.on('pull_request.reopened', async context => {
-    if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
-      await checkPullRequestLabelsModule.checkChangelogLabel(context);
-      // Prevent user from reopening the PR.
-      await checkPullRequestBranchModule.checkBranch(context);
-      await checkWIPModule.checkWIP(context);
+    if (
+      whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
+      await runChecks(context, 'reopened');
     }
   });
 
   oppiabot.on('pull_request.labeled', async context => {
-    if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
-      await checkPullRequestLabelsModule.checkAssignee(context);
+     if (
+      whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
+      await runChecks(context, 'labeled');
     }
   });
 
   oppiabot.on('pull_request.synchronize', async context => {
-    if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
+    if (
+      whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
       // eslint-disable-next-line no-console
       console.log(' PR SYNC EVENT TRIGGERED..');
-      await checkMergeConflictsModule.checkMergeConflictsInPullRequest(
-        context, context.payload.pull_request);
+      await runChecks(context, 'synchronize');
     }
   });
 
   oppiabot.on('pull_request.closed', async context => {
-    if (whitelistedAccounts.includes(context.repo().owner.toLowerCase()) &&
+     if (

```

Comment:
I suggest moving the whitelisted accounts to a helper method to make this if & other checks a bit shorter so that they don't have to be wrapped like this.

Reply:
Done.